content/post/101-rust-ipfs-unixfs.md

```diff
@@ -0,0 +1,152 @@
+---
+date: 2020-06-29
+url: /2020-06-29-rust-ipfs-unixfs/
+header_image: rust-ipfs-crab.png
+title: Rust IPFS UnixFS Support Lands
+author: Joonas Koivunen and Mark Robert Henderson
+tags: Rust, IPFS, UnixFS
+---
+
+UnixFS reading, or _exporting_, has landed in [Rust IPFS]! Funded by a
+Protocol Labs [devgrant], the main deliverables were the [`ipfs-unixfs`] crate,
+and the HTTP APIs `/get`, and `/cat`.
+
+[devgrant]: https://github.com/ipfs/devgrants/tree/master/open-grants/ipfs-rust/phase-2
```

Comment:
these don't show up in the preview right now - not sure they'll show up in the HTML either. Are you trying to make these links?

Reply:
Aka - are you explaining how unixfs works? how to use it in rust-ipfs? something else? By targeting your header titles for the particular user, you'll make them more descriptive and actionable, and also do a better job breaking up content into easily consumable bites

Reply:
> these don't show up in the preview right now - not sure they'll show up in the HTML either. Are you trying to make these links?

I at least tested this via `npm install && npm start` locally and they do show up in the HTML. I believe this is just an alternative way of creating links in markdown to not mess with the flow of the paragraph. Bummer it doesnt show up in the GH preview :(

![image](https://user-images.githubusercontent.com/106148/86178481-6b1ceb80-baf6-11ea-9deb-07cde4eacdf3.png)